stratum/hal/lib/barefoot/bfrt_constants.h

```diff
@@ -67,6 +70,14 @@ constexpr int kMaxQueuesPerPort = 32;
 
 constexpr absl::Duration kDefaultSyncTimeout = absl::Seconds(1);
 
+// P4Runtime Translation.
+constexpr int32 kTnaPortIdBitWidth = 9;
+constexpr char kUriTnaPortId[] = "tna/PortId_t";
+const absl::flat_hash_map<std::string, int32> kUriToBitWidth = {
+    {kUriTnaPortId, kTnaPortIdBitWidth}};
+constexpr uint32 kSdnTnaRecirculationPortBase = 0xFFFFFF00;
+constexpr uint32 kTnaRecirculationPortBase = 0x44;
```

Comment:
I think those are not used outside the translator, thus could be local to that class.